HW_2/task_1.py

```diff
@@ -0,0 +1,22 @@
+#"""
+#Напишите программу, которая принимает на вход вещественное число и показывает сумму его цифр.
+
+#Ввод: значение типа <float>
+#Вывод: значение типа <int>
+
+#Примеры:
+#6782.0
+#23
+
+#0.56
+#11
+#"""
+number = input('Введите вещественное число: ')
+
+def sum (number):
+    sum = 0
+    for i in number:
+        if i != '.':
+            sum = sum + int(i)
```

Comment:
```suggestion
            sum += int(i)
```
Здесь произойдет ошибка. Если у вас будет отрицательное число.